ios/Classes/SwiftPolarPlugin.swift

```diff
@@ -300,11 +297,9 @@ public class SwiftPolarPlugin:
 
     _ = api.fetchExercise(identifier, entry: entry).subscribe(
       onSuccess: { data in
-        guard let data = jsonEncode(PolarExerciseDataCodable(data))
```

Comment:
Again please don't make changes that are unnecessary. This is more of a style choice than anything. Same for the guard statements below.